subsys/usb/class/usb_dfu.c

```diff
@@ -63,6 +63,10 @@ LOG_MODULE_REGISTER(usb_dfu);
 #define USB_DFU_MAX_XFER_SIZE		CONFIG_USB_DFU_MAX_XFER_SIZE
 #endif
 
+static struct k_work dfu_work;
+u8_t buf[USB_DFU_MAX_XFER_SIZE];
+u32_t dfu_len, dfu_state;
```

Comment:
static? I would use dfu_worker_len and dfu_worker_state. Or would it not be better to put it inside dfu_data_t and call them worker_len and worker_state?

Reply:
packed all variables related to a worker thread in a structure.